templates/galaxy/config/global_host_filters.py.j2

```diff
@@ -85,7 +87,8 @@ DOMAIN_SECTIONS = {
     'metabolomics': ['metabolomics'],
     'ml': ['machine_learning'],
     'annotation': GENERAL_NGS_SECTIONS + ['annotation', 'apollo', 'ncbi_blast', 'assembly', 'variant_calling', 'fetch_sequences___alignments', 'ontology', 'emboss', 'evolution', 'genome_diversity', 'multiple_alignments', 'graph_display_data'],
-
+    'materials': ['materials_science'],
```

Comment:
Not entirely sure what this is meant to be referencing - tool shed category?

Reply:
Tool categories ... Defined in the tool_conf.xml file

Reply:
Aha, of course. Added.